src/strings/RemoveDuplicates.java

```diff
@@ -0,0 +1,33 @@
+package strings;
+
+import java.util.Scanner;
+
+public class RemoveDuplicates {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
```

Comment:
String result = "";

        for (int i = 0; i < string.length(); ++i) {
            char currentElement = string.charAt(i);
            if (!result.contains(Character.toString(currentElement))) {
                result = result + currentElement;
            }
        }
        System.out.println("After removing duplicates characters the new string is: "+ result);

erexeqic mekic copy areci)) bayc gaxapary nuynna))
qo aracov el klni uxaki qonov 2 hat for a irar  mej, espes aveli arag klni